game/transfers.py

```diff
@@ -774,3 +780,13 @@ def order_airlift_assets_at(self, control_point: ControlPoint) -> None:
         self.game.coalition_for(self.player).add_procurement_request(
             AircraftProcurementRequest(control_point, FlightType.TRANSPORT, gap)
         )
+
+    def transfer_for_flight(self, flight: Flight) -> Optional[TransferOrder]:
```

Comment:
It is used from the `FlightGroupConfigurator` to determine the transfer for Transport Flights so that we can add the information about the Cargo to the MissionData which then is used to add the cargo type as new crate for CTLD :)